src/components/Modals/SlotMenu.vue

```diff
@@ -42,25 +41,24 @@ import TeleportModal from '@/components/Modals/TeleportModal.vue';
 export default defineComponent({
   components: { TeleportModal },
   props: {
-    modelValue: { required: true, type: Boolean },
     position: { type: Object as PropType<{ x: number; y: number }>, required: true },
   },
   emits: {
     'open-delete-slot-modal': () => true,
     'open-edit-slot-modal': () => true,
-    'update:modelValue': (value: boolean) => typeof value === 'boolean',
+    'close-slot-menu': (value: boolean) => typeof value === 'boolean',
   },
   methods: {
     closeCurrentModal() {
-      this.$emit('update:modelValue', false);
+      this.$emit('close-slot-menu', false);
     },
     openDeleteSlotModal() {
       this.$emit('open-delete-slot-modal');
-      this.$emit('update:modelValue', false);
+      this.$emit('close-slot-menu', false);
```

Comment:
One thing we do in a bunch of other modals is call closeCurrentModal() from other functions that need to close the modal instead of duplicating the emit. Nit but might be good to be consistent and avoid calling the emit so many times!